scipy/stats/tests/test_stats.py

```diff
@@ -1177,11 +1177,17 @@ def test_structured_arrays(self):
 
 
 class TestMode(TestCase):
+    def test_empty(self):
+        data1 = []
+        vals = stats.mode(data1)
+        assert_equal(vals[0], np.array([]))
+        assert_equal(vals[1], np.array([]))
+
     def test_basic(self):
         data1 = [3,5,1,10,23,3,2,6,8,6,10,6]
         vals = stats.mode(data1)
-        assert_almost_equal(vals[0][0],6)
-        assert_almost_equal(vals[1][0],3)
+        assert_equal(vals[0][0],6)
+        assert_equal(vals[1][0],3)
```

Comment:
in the lines you changed, it would be nice to immediate make them PEP8-compliant (spaces after commas)